dataprep/eda/correlation/compute.py

```diff
@@ -76,42 +79,73 @@ def compute_correlation(
             k is None
         ), "value_range and k cannot be present in both"
 
-        df = df.select_dtypes(NUMERICAL_DTYPES)
+        df_num = df.select_dtypes(NUMERICAL_DTYPES)
+
         if len(df.columns) == 0:
             return Intermediate(visual_type=None)
 
-        data = df.to_dask_array()
+        data_num = df_num.to_dask_array()
         # TODO Can we remove this? Without the computing, data has unknown rows so da.cov will fail.
-        data.compute_chunk_sizes()
+        data_num.compute_chunk_sizes()
 
-        cordx, cordy, corrs = correlation_nxn(data)
-        cordx, cordy = dd.from_dask_array(cordx), dd.from_dask_array(cordy)
-        columns = df.columns
+        cordx_num, cordy_num, corrs_num = numerical_correlation_nxn(data_num)
+        cordx_num, cordy_num = (
+            dd.from_dask_array(cordx_num),
+            dd.from_dask_array(cordy_num),
+        )
+        columns_num = df_num.columns
+        cordx_phik, cordy_phik, corrs_phik, columns_phik = phik_correlation_nxn(df_org)
```

Comment:
what does the 'columns_phik' means?

Reply:
columns_phik means the columns that are selected to calculate the phik correlation

Reply:
As phi_k supports all types of columns, how the columns are selected? Is it because of efficiency issue?

Reply:
Yes, it is because of efficiency issue. Too many columns will make the calculation of Phik correlation become slowly.